partner/models.py

```diff
@@ -0,0 +1,64 @@
+from django.db import models
+
+class Counselor(models.Model):
+    name              = models.CharField(max_length=50)
+    gender            = models.CharField(max_length=10)
+    level             = models.ForeignKey('Level', on_delete=models.CASCADE)
+    counsel_count     = models.IntegerField(default=0)
+    introduction      = models.CharField(max_length=1000)
+    is_counsel_gt_150 = models.IntegerField(default=0)
```

Comment:
is 인데 boolean 아닌가요 ? 
```suggestion
    is_counsel_gt_150 = models.IntegerField(default=0)
```

Reply:
그리고 is_ counsel_count_gt_150 이라고 하는게 나을것같습니다.